sdk/core/core-httpclients/package.json

```diff
@@ -0,0 +1,87 @@
+{
+  "name": "@azure/core-httpclients",
```

Comment:
any better package names?

Reply:
Ramblings here:
@azure/core-primitives if we expect to expose more lower level primitives
@azure/core-util - maybe these are utils and can be part of the exisitng package?

Reply:
We'd like this to be an optional package. I am using core-http-client-xhr for now. Fine for a preview/beta?